containers/base/Dockerfile

```diff
@@ -123,20 +124,6 @@ RUN echo "deb-src http://ftp.us.debian.org/debian testing main" >> /etc/apt/sour
     rm -rf /usr/share/i18n/locales/* && \
     cd /
 
-# Install ProtoBuf, TensorFlow
```

Comment:
Those are relatively "big" dependencies. Someone using pydatalab for just working with BigQuery for example shouldn't have to bring them down as well.